de.metas.fresh/de.metas.fresh.base/src/main/java/de/metas/fresh/product/process/ExportProductSpecifications.java

```diff
@@ -69,24 +71,25 @@ public IExportDataSource createDataSource()
 		builder.addWhereClause("1=1", new Object[] {});
 		builder.addOrderBy("productValue");
 
-		builder.addField("Artikelbezeichnung", "productName");
-		builder.addField("Sachbezeichnung", "CustomerLabelName");
-		builder.addField("Obligatorische Zusatzangaben", "additional_produktinfos");
-		builder.addField("Artikelnummer", "productValue");
-		builder.addField("EAN Code CU", "UPC");
-		builder.addField("Nettogewicht", "weight");
-		builder.addField("Produktionsland", "country");
-		builder.addField("Verpackungsmaterial", "piName");
-		builder.addField("Anzahl CU/TU", "piQty");
-		builder.addField("Haltbarkeit ab Produktionstag/Abpacktag", "guaranteedaysmin");
-		builder.addField("Lager- und Transporttemperatur", "warehouse_temperature");
-		builder.addField("Produktebeschrieb", "productDecription");
-		builder.addField("Stückliste Komponente", "componentName");
-		builder.addField("Zutaten", "componentIngredients");
-		builder.addField("%- Anteil", "qtybatch");
-		builder.addField("Allergene", "allergen");
-		builder.addField("Nährwerte", "nutritionName");
-		builder.addField("Nährwerte menge", "nutritionqty");
+		
+		builder.addField(msgBL.translate(getCtx(), "ProductName"), "productName");
+		builder.addField(msgBL.translate(getCtx(), "CustomerLabelName"), "CustomerLabelName");
+		builder.addField(msgBL.translate(getCtx(), "Additional_produktinfos"), "additional_produktinfos");
+		builder.addField(msgBL.translate(getCtx(), "ProductValue"), "productValue");
+		builder.addField(msgBL.translate(getCtx(),"UPC"), "UPC");
+		builder.addField(msgBL.translate(getCtx(),"NetWeight"), "weight");
+		builder.addField(msgBL.translate(getCtx(),"Country"), "country");
+		builder.addField(msgBL.translate(getCtx(),"IsPackagingMaterial"), "piName");
+		builder.addField(msgBL.translate(getCtx(),"NumberOfEvents"), "piQty");
```

Comment:
hi @cristinamghita  is there really an AD_Message or AD_Element `NumberOfEvents` that translated to "Anzahl CU/TU"